ui/lib/apps/Statement/utils/useStatementTableController.ts

```diff
@@ -112,7 +112,7 @@ export default function useStatementTableController(
     [queryOptions, allTimeRanges]
   )
 
-  const [loadingStatements, setLoadingStatements] = useState(false)
+  const [loadingStatements, setLoadingStatements] = useState(true)
```

Comment:
I remember it was `true` before then I modified it to `false`, you can see the commit history to understand why it was changed.

Reply:
![image](https://user-images.githubusercontent.com/1284531/106700619-2aeb9180-6620-11eb-90af-58447301f49b.png)

Reply:
Ok, I'll check it.

Reply:
This is used in combination with cache item. Wow, almost made a mistake.

Reply:
But I think the semantics of the previous code is not clear enough and may cause bug.

Reply:
OK, I will have a test when I am free.